kube-client/src/error.rs

```diff
@@ -75,6 +71,12 @@ pub enum Error {
     #[error("openssl tls error: {0}")]
     OpensslTls(#[source] crate::client::OpensslTlsError),
 
+    /// Errors from Rustls TLS
+    #[cfg(feature = "rustls-tls")]
+    #[cfg_attr(docsrs, doc(feature = "rustls-tls"))]
```

Comment:
think you need to:

```suggestion
    #[cfg_attr(docsrs, doc(cfg(feature = "rustls-tls")))]
```

got a rustdoc error from a similar thing in the previous pr

Reply:
Ah, thanks. I probably copied and pasted what you fixed in that PR :|